pkg/compiler/rego/rego.go

```diff
@@ -0,0 +1,108 @@
+package compiler_rego
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infobloxopen/seal/pkg/ast"
+	"github.com/infobloxopen/seal/pkg/compiler/error"
+)
+
```

Comment:
this package should import the compiler backpage. then it should have an init() {} that registers that backend language contructor

Reply:
will fix

Reply:
done